### ISSUE TITLE
Apply to be a member 申請加入團隊 / [Jun Feng]

### ISSUE THREAD

**Issue by Jun233333** [UMHelper/Feedback-and-Join-Us#20]
* I am _Jun Feng__, year _Three__ student majoring in _Statistics and  Data Science_.
* Having skills in __Data Analysis( R and Python), C, EXCEL, PPT__. 
* Learning _Analysis Skills and Communication Skills
* Desired team: Marketing
* Email: db92849@umac.mo
* Here is my CV: https://drive.google.com/file/d/1-2-gj5-7BMDKD-2XiZm_iAGkiXHWqgM-/view?usp=drivesdk

**Comment by poplar-xirolan** [comment]
Hello! I have a small question. It seems that you are going to graduate next year. Can you continue to study in UM after you get your bachelor's degree? If so, it would be of significant help to us since some of our team members are going to graduate next year, and may not be able to continue maintaining this project as frequently as before.

**Comment by Jun233333** [comment]
Hi！Thank you for your reply！For the questions you rised, the following is my viewpoints.There would be a great possibility that I will continue to study in UM.

Where I am going to study depends on many factors, such as money and epidemic. Studying in UM, I can apply for the teaching assistent, which would pay me 7000 MOP a month. What's more, there is no safety problem in Macau while there is a growing unrest abroad. Due to my GPA and other conditions, and compared to some possible options, it seems that UM is a relatively good place to continue studying. I would study in UM if there is no big change and good luck after I graduate.

But of course, when I luckly get the satisfactory offer in the coming application season and my family is willing to support the high cost( it's expensive to continue to study, right？), I would study in other university. For the satisfactory school, it would be a university I have litter chance of getting it.

In conclusion, there is no need to worry about whether I would continue to study in UM because it's indeed a great choose for me.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by BoxMars** [comment]
do u have some experience in web development using Python, Java or PHP?

**Comment by Jun233333** [comment]
Thank you for your reply！I have no experience in web development but what I apply for is the market.

However, if there is no position in market, I have confidence in learning the web development as I have experience in use of python and other computer language, such as C language and R language.

Looking forward to your reply！

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
